IS/publiccode.yml

```diff
@@ -0,0 +1,109 @@
+# This repository adheres to the publiccode.yml standard by including this
+# metadata file that makes public software easily discoverable.
+# More info at https://github.com/italia/publiccode.yml
+
+publiccodeYmlVersion: '0.2'
```

Comment:
We should add maintenance section -> Below is what it says in their doc.

```
1.1.22. Section maintenance
 see actions
This section provides information on the maintenance status of the software, useful to evaluate whether the software is actively developed or not.1.1.22.1. Key maintenance/type
Type: enumerate
Presence: mandatory
Values: "internal", "contract", "community", "none"
This key describes how the software is currently maintained.
internal - means that the software is internally maintained by the repository owner;
contract - means that there is a commercial contract that binds an entity to the maintenance of the software;
community - means that the software is currently maintained by one or more people that donate their time to the project;
none - means that the software is not actively maintained.
```

So we have to mention our product as "contract" I believe

Reply:
```
1.1.22.3. Key maintenance/contacts
Type: List of Contacts (see below)
Presence: mandatory (if maintenance/type is internal or community)
One or more contacts maintaining this software.This key describes the technical people currently responsible for maintaining the software. All contacts need to be a physical person, not a company or an organisation. If somebody is acting as a representative of an institution, it must be listed within the affiliation of the contact.In case of a commercial agreement (or a chain of such agreements), specify the final entities actually contracted to deliver the maintenance. Do not specify the software owner unless it is technically involved with the maintenance of the product as well.
```

Above is also in the doc we may have to discuss, how we do it

Reply:
Didn't change as per the decision taken over the call

Reply:
Hi thouught we are to add maintenance  with contract